back/app/services/local_project_copy_service.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+# Copies a project within a tenant.
+class LocalProjectCopyService
+  def copy(source_project)
+    new_title_multiloc = add_suffix_to_title(source_project.title_multiloc)
+
+    options = {
+      local_copy: true,
+      include_ideas: false,
+      anonymize_users: false,
+      new_title_multiloc: new_title_multiloc,
+      timeline_start_at: Time.now.to_s,
+      new_publication_status: 'draft'
+    }
+
+    template = AdminApi::ProjectCopyService.new.export source_project, **options
+    folder_id = ProjectFolders::Folder.find(source_project.folder_id) if source_project.folder_id
+    copied_project = AdminApi::ProjectCopyService.new.import(template, folder: folder_id, local_copy: true)
+
+    copy_project_visibility_permission_groups(source_project, copied_project)
+
+    # Copy actions groups_permissions of project or project phases. For example, groups that can 'Vote on ideas'.
```

Comment:
This comment duplicates the method name and the comments inside. I would remove it.

(up to you)